pkg/build/info.go

```diff
@@ -112,8 +112,8 @@ func BinaryVersion() string {
 func VersionForURLs() string {
 	if parsedVersionTxt.IsPrerelease() {
 		phaseAndOrdinal := parsedVersionTxt.Format("%P.%o")
-		// builds prior to "alpha.1" use 'dev' in their URLs
-		if phaseAndOrdinal < "alpha.1" {
+		// builds prior to "rc.1" use 'dev' in their URLs
+		if phaseAndOrdinal < "rc.1" {
```

Comment:
actually...  rc.1 is also a test release (also shipped before GA) -- so maybe conditional should be more like below?

```
if phaseAndOrdinal <= "rc.5" {
...
```

Reply:
I was not sure what version to use. "rc.5" sounds extreme, maybe better to use "dev" for all pre-release versions then? At least it would be clearer comparing to "rc.5".

Reply:
It's even simpler: `if parsedVersionTxt.IsPrerelease() || parsedVersionTxt.IsCustomOrNightlyBuild()`

Reply:
> It's even simpler: if parsedVersionTxt.IsPrerelease() || parsedVersionTxt.IsCustomOrNightlyBuild()
nice! works for me - thank you!